test.js

```diff
@@ -125,6 +125,20 @@ describe('superagent mock', function() {
         });
     });
 
+    it('should throw error with unmocked requests when useMocksOnly enabled', function(done) {
+        mock.enableUseMocksOnly();
+
+        should.throws(function() {
+            request
+                .get('http://example.com')
+                .query({ foo: 'bar' })
+                .end(function(err, res) {});
+        }, /Unmocked url/);
+        done();
+
+        mock.disableUseMocksOnly();
```

Comment:
Shouldn't the clean up code happen before we call `done()`. My assumption was that once `done()` was called, the next test starts running.